tdm-be/src/templates/processing-error.njk

```diff
@@ -4,6 +4,10 @@ Vous trouverez dans ce mail un rapport d'erreur du traitement {{ processingId }}
 
 {{ errorMessage }}
 
+Vous pouvais retrouvé une description de la cause possible de l'erreur sur le lien ci-dessous :
+
+https://github.com/Inist-CNRS/tdm-factory/blob/main/ERROR_CODE_FR.md
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** Let's make more MarkDown interpreters work

‏
```suggestion
<https://github.com/Inist-CNRS/tdm-factory/blob/main/ERROR_CODE_FR.md>
```

Reply:
I don't thinks this work in text email :/